base/record_replay.h

```diff
@@ -121,8 +121,8 @@ int PointerId(const void* ptr);
 void* IdPointer(int id);
 
 void OnEvent(const char* aEvent, bool aBefore);
-void OnMouseEvent(const char* kind, size_t clientX, size_t clientY);
-void OnKeyEvent(const char* kind, const char* key);
+void OnMouseEvent(const char* kind, size_t clientX, size_t clientY, bool synthetic = false);
+void OnKeyEvent(const char* kind, const char* key, bool synthetic = false);
```

Comment:
can we instead make this parameter required and fix up the spots where it's called?

Reply:
We could, but not sure why?

Reply:
just general dislike of default parameters.  normally when defaults are added for newly added parameters it's because there's a boatload of existing call sites that we don't also want to update, but there won't be many of these, will there?